client/core/trade.go

```diff
@@ -1599,7 +1599,8 @@ func (c *Core) redeemMatchGroup(t *trackedTrade, matches []*matchTracker, errs *
 	// Send the transaction.
 	redeemWallet, redeemAsset := t.wallets.toWallet, t.wallets.toAsset // this is our redeem
 	coinIDs, outCoin, fees, err := redeemWallet.Redeem(&asset.RedeemForm{
-		Redemptions: redemptions,
+		Redemptions:   redemptions,
+		FeeSuggestion: c.feeSuggestion(t.dc, t.dc.bookie(t.mktID), redeemWallet.AssetID, !t.Trade().Sell),
```

Comment:
The `feeSuggestion` call can make a request to the server.  That's a problem in `redeemMatchGroup`.  We split out the actual init and redeem requests to make them asynchronous (`sendInitAsync` and `sendRedeemAsync`) because of the trouble this can cause, but ideally we'd have no requests at all for this. All of the automated (tick or block based) actions need to be snappy and not have the potential to block.  Is there any way to avoid this here, either changing `feeSuggestion` or ensuring there is already a value ready somewhere for this?